### ISSUE TITLE
SoundStream

### ISSUE THREAD

**Issue by Dannynis** [lucidrains/audiolm-pytorch#120]
Hey, I'm trying to overfit the SoundStream model on 10 examples using the most basic setup:
 - no attentions
 - only reconstruction loss
 https://colab.research.google.com/drive/1pEA1vQwxNnrj7g6B_zOi4laEc82VMFB8?usp=sharing
But still couldn't managed to make it work....
Any suggestions on why is it happening ?
(p.s disabling the discriminator causes error at the trainer - .item() should be discarded)

Thank you !

**Comment by lucidrains** [comment]
@Dannynis hey, would recommend you move this to discussions

multiple people in industry (and some in academia) have already gotten good results training soundstream
